recipes/osx-sysroot/activate-mdt.sh

```diff
@@ -0,0 +1,47 @@
+if [ -z "$MACOSX_DEPLOYMENT_TARGET" ]; then
+    export MACOSX_DEPLOYMENT_TARGET="@MACOSX_DEPLOYMENT_TARGET@"
+else
+    export CONDA_SYSROOT_@PLATFORM@_BACKUP_MACOSX_DEPLOYMENT_TARGET="${MACOSX_DEPLOYMENT_TARGET}"
+    export MACOSX_DEPLOYMENT_TARGET="@MACOSX_DEPLOYMENT_TARGET@"
+fi
+
+if [ -z "${CMAKE_ARGS}" ]; then
+    export CMAKE_ARGS="-DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET}"
+else
+    export CONDA_SYSROOT_@PLATFORM@_BACKUP_CMAKE_ARGS="${CMAKE_ARGS}"
+    export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET}"
+fi
+
+if [ -z "${CPPFLAGS}" ]; then
+    export CPPFLAGS="-mmacosx-version-min=${MACOSX_DEPLOYMENT_TARGET}"
+else
+    export CONDA_SYSROOT_@PLATFORM@_BACKUP_CPPFLAGS="${CPPFLAGS}"
+    export CPPFLAGS="${CPPFLAGS} -mmacosx-version-min=${MACOSX_DEPLOYMENT_TARGET}"
+fi
+
+if [ ! -z "$MACOSX_SDK_VERSION" ]; then
+    sdk_parts=${MACOSX_SDK_VERSION//./' '}
+    sdk_parts=($sdk_parts)
+    mdt_parts=${MACOSX_DEPLOYMENT_TARGET//./' '}
+    mdt_parts=($mdt_parts)
```

Comment:
Is there a reason to do it in two steps?
```suggestion
    # read version into array (which uses space separation, so replace the dots)
    sdk_parts=(${MACOSX_SDK_VERSION//./' '})
    mdt_parts=(${MACOSX_DEPLOYMENT_TARGET//./' '})
```